doc/source/reference/security.rst

```diff
@@ -0,0 +1,60 @@
+NumPy security considertions and reporting
```

Comment:
typo in considerations

Reply:
More importantly: this is a little too long for the left sidebar in the html docs. Also forward-looking when we address gh-21713, how about just naming this section `NumPy Security`?